src/error.rs

```diff
@@ -624,7 +624,7 @@ pub trait ToDiagnostic<FileId> {
     fn to_diagnostic(
         &self,
         files: &mut Files<String>,
-        contract_id: Option<FileId>,
+        contracts_id: &Option<Vec<FileId>>,
```

Comment:
This thing shouldn't be called `contracts_id` anymore, probably, now that it's the file ids of all modules of the stdlib (it was named like that because it used to be the `FileID` of the `contact` module).